src/utils/common.ts

```diff
@@ -110,6 +110,26 @@ export function errorHasValidationKey(error: any, expectedKey: string): boolean
   return error.validation?.some(({ key }: any) => expectedKey === key);
 }
 
+/**
+ * Create list of callbacks
+ */
+export function createCallbackStorage<T extends(...args: any) => any>() {
```

Comment:
I think the name is a bit misleading, I would suggest `createCallbackList` or `createCallbackRegistry` or `createCallbackArray`

Reply:
Totally agree, I overused the word "storage". Changed to `Registry`.